.env.example

```diff
@@ -14,6 +14,8 @@ DEPLOYER_PRIVATE_KEY=
 #ARCHIVE_NODE_ethereum=http://127.0.0.1:1248
 #ARCHIVE_NODE_opbnbtestnet=https://opbnb-testnet.nodereal.io/v1/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_opbnbmainnet=https://opbnb-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
+#ARCHIVE_NODE_arbitrumOne=https://arb1.arbitrum.io/rpc
```

Comment:
```suggestion
#ARCHIVE_NODE_arbitrumOne=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/arbitrum-nitro/
```

To be confirmed

Reply:
That's a public url, we can use any one.

Reply:
Done https://github.com/VenusProtocol/venus-protocol/pull/456/commits/070012779c9bf4a5a306e9cdc4133c986e450b95